### DISCUSSION TITLE
(Feature Request) Syntax High Lighting for Batch scripts

### DISCUSSION THREAD

**Discussion by Br0kenSilos** [TriliumNext/Notes#829] [Ideas]
Would it be possible to add syntax highlighting for Windows/CMD batch scripts in either or both the Code note type and the code snippets?

I do still write a lot of long and semi complicated batch scripts. Usually I write these in VS Code because the syntax highlighting there is really helpful to me. As you can see in the examples below, it makes a huge difference in readability. But I also like to store my batch and other scripts  and do version control in my Trilium.  

I do realize that very few people write batch scripts these days so it may not be worth the effort. But IF it's simple to do, I would really love it. 

![notepad](https://github.com/user-attachments/assets/47c17351-af00-4109-a745-674a2d72723d)
![vscode](https://github.com/user-attachments/assets/71cc54a5-346c-4479-8c16-7f73d071e96f)

**Comment by eliandoran** [comment]
Converted to issue and solved: https://github.com/TriliumNext/Notes/issues/830
